pipeline-examples/archive-build-output-artifacts/ArchiveBuildOutputArtifacts.groovy

```diff
@@ -0,0 +1,18 @@
+// This shows a simple example of how to archive the build output artifacts.
+node {
+    stage "Create build output"
+    
+    // Make the output directory.
+    sh "mkdir -p output"
+
+    // Write an useful file, which is needed to be archived.
+    writeFile file: "output/usefulfile.txt", text: "This file is useful, need to archive it."
+
+    // Write an useless file, which is not needed to be archived.
+    writeFile file: "output/uselessfile.md", text: "This file is useless, no need to archive it."
+
+    stage "Archive build output"
+    
+    // Archive the build output artifacts.
+    archive excludes: 'output/*.md', includes: 'output/*.txt'
```

Comment:
`archive` is (unfortunately) deprecated in favor of `archiveArtifacts` now, which supports artifact fingerprinting.

I think this can be merged once it is updated to reflect that change